person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+
+  def can_use_services?
+    of_age? || parent_permission == true
+  end
+
+  private
+
+  def of_age?
+    age >= 18
+  end
```

Comment:
- [ ] Kindly add `@` to the age here to ensure we are using the instance variable declared.